scripts/package/deb-after-install.sh

```diff
@@ -0,0 +1,109 @@
+#! /bin/bash
+INSTALL_PREFIX=/data
+
+# ensure base directories owned
+chown socorro /var/log/socorro
+chown -R www-data.socorro $INSTALL_PREFIX/socorro/webapp-django/static/CACHE
+
+# crond doesn't like files with executable bits, and doesn't load
+# them.
+chmod 644 /etc/cron.d/socorro
+
+# Link the local Django settings to the distributed settings.
+ln -fs /etc/socorro/local.py $INSTALL_PREFIX/socorro/webapp-django/crashstats/settings/local.py
+
+# change mod_wsgi to python 2.6
+ln -fs /usr/lib/apache2/modules/mod_wsgi.so-2.6 /usr/lib/apache2/modules/mod_wsgi.so
```

Comment:
We're planning on dropping the 2.6 requirement soon... you should be able to use 2.7 now and not have to deal with 2.6 if you want.